dateutil/test/test_isoparser.py

```diff
@@ -342,24 +335,8 @@ def __make_date_examples():
 
     dates_w_day_fmts = ('%Y%m%d', '%Y-%m-%d')
     o = it.chain(o, it.product(dates_w_day, dates_w_day_fmts))
-
     return list(o)
 
-
-@pytest.mark.parametrize('d,dt_fmt', __make_date_examples())
```

Comment:
Why was this test removed?

Reply:
I removed this test because it's testing date strings in the format "YYYY-MM" and I thought that format was invalid as well. I'll add it back.
Just to make sure I've understood the problem correctly, date strings in the format "YYYYMM" should raise a ```ValueError``` right?

Reply:
Yes.